app/models/framework_question.rb

```diff
@@ -22,7 +22,7 @@ class FrameworkQuestion < VersionedModel
   has_many :framework_responses
   has_and_belongs_to_many :framework_nomis_codes, autosave: true
 
-  def build_responses(question: self, assessmentable:, questions:, previous_responses: {})
+  def build_responses(assessmentable:, questions:, question: self, previous_responses: {})
```

Comment:
Is this something to do with alphabetical order of named params?

Reply:
yep, for named params the order isn't imprtant, thus Rubocop decrees they must be alphabetical